### DISCUSSION TITLE
Crash when running the provided tutorial code.

### DISCUSSION THREAD

**Discussion by CatoConroy** [alphaparrot/ExoPlaSim#12] [Q&A] [answered]
![image](https://github.com/alphaparrot/ExoPlaSim/assets/88071894/e5afcb46-2798-47fc-b8da-f86b044967e5)

Above I provided the output when running the program. Code used is this:

`import exoplasim as exo
mymodel = exo.Model(workdir="mymodel_testrun",modelname="mymodel",resolution="T21",layers=5,precision=4,ncpus=8)
mymodel.configure()
mymodel.run(years=10,crashifbroken=True)
mymodel.finalize("mymodel_output")
`

I believe I installed all prerequisites as well. Any help would be greatly appreciated.

**Comment by brianthomas-washburn** [comment, marked as answer]
It looks like the compiler was not able to build the executable.  I would check to make sure that you can compile f90 source and that mpi is correctly installed.
